sw/vendor/meson.build

```diff
@@ -9,3 +9,59 @@ vendor_coremark_base_files = files([
   'eembc_coremark/core_state.c',
   'eembc_coremark/core_util.c',
 ])
+
+# Googletest and Googlemock Build Instructions
+thread_dep = dependency('threads')
+sw_vendor_gtest_dir = 'google_googletest'
+
+# Begin Based on https://github.com/mesonbuild/gtest
```

Comment:
This URL takes me to the master branch which doesn't have any `meson.build` files. Can you add https://github.com/mesonbuild/gtest/blob/1.10.0/googlemock/meson.build and https://github.com/mesonbuild/gtest/blob/1.10.0/googletest/meson.build to the conment as well?

Reply:
New comment links directly to the `1.10.0` tree.